locales/en/messages.json

```diff
@@ -6220,6 +6220,14 @@
         "message": "<span class=\"message-negative\">Error</span> while saving the VTX Config file",
         "description": "Message in the GUI log when the VTX Config file is saved"
     },
+    "vtxSavedLuaFileOk": {
+        "message": "lua VTX Config file <span class="message-positive">saved",
+        "description": "Message in the GUI log when the lua VTX Config file is saved"
+    },
+        "vtxSavedLuaFileKo": {
+        "message": "<span class="message-negative">Error while saving the lua VTX Config file",
```

Comment:
The strings are cut, please complete them ;)

Reply:
You need to close the </span> tag

Reply:
Missing \ lost in copy paste